doc/how-to/delete_booking_or_premises_remotely.md

```diff
@@ -0,0 +1,13 @@
+# How to delete a Booking or Premises remotely
```

Comment:
I initially overlooked voids in this equation and if we should prevent delete the premises if they exist.

Thinking about it now I think think not. Given this script will be used in edge cases to catch admin errors and the check for bookings would be the crucial record of accommodation.

Perhaps worth noting the extra behaviour here in our docs though?

- If the premises has at least one booking, it won't delete
- If the premises has any amount of voids, they will be included in the deletion